test/controllers/groups_controller_test.rb

```diff
@@ -51,16 +51,16 @@ class GroupsControllerTest < ActionDispatch::IntegrationTest # rubocop:disable M
     sign_in users(:john_doe)
 
     group = groups(:group_one)
-    patch group_path(group), params: { group: { name: 'New Group Name' } }
-    assert_redirected_to group_path(group)
+    patch group_path(group, format: :turbo_stream), params: { group: { name: 'New Group Name' } }
```

Comment:
We should move the `format: :turbo_stream` to the params hash before the closing curly brace as we did for the projects_controller_test. Just to keep it consistent

Reply:
Please see e85abc419e4970d1f29684887d4744e5f8f1a58e.